src/include/function/table/bind_data.h

```diff
@@ -69,29 +59,5 @@ struct KUZU_API TableFuncBindData {
     std::vector<storage::ColumnPredicateSet> columnPredicates;
```

Comment:
Not related to this PR but I think you can also move columnSkip&Preidicate out of the base class?

Reply:
I think we should propogate the projection push down to all table functions so i would keep these two fields there